dotcom-rendering/src/components/FooterReaderRevenueLinks.importable.tsx

```diff
@@ -0,0 +1,154 @@
+import { css } from '@emotion/react';
```

Comment:
This file replaces `ReaderRevenueLinks.importable.tsx`, deleting much of the code.
In particular, it deletes code for calling out to SDC (the marketing API), which is not used for the footer links.